src/main/java/io/github/christophermanahan/carnitas/Response.java

```diff
@@ -0,0 +1,5 @@
+package io.github.christophermanahan.carnitas;
+
+public interface Response {
+  byte[] bytes();
```

Comment:
Why `bytes`?  I feel like this name is not very informative. What are we doing here? What is the intention? How can we update the name to express that intention?